cmake/modules/ReadVersion.cmake

```diff
@@ -1,7 +1,7 @@
 # Read rocksdb version from version.h header file.
 
 function(get_rocksdb_version version_var)
-  file(READ "${CMAKE_SOURCE_DIR}/include/rocksdb/version.h" version_header_file)
+  file(READ "${CMAKE_CURRENT_SOURCE_DIR}/include/rocksdb/version.h" version_header_file)
```

Comment:
why?

Reply:
Since I'm trying to build a distributed storage system based on rocksdb, and my codebase directory structure just like this.

```
/path/to/rocskdb_xxx$ tree -L 1
.
|-build.sh
|-my_code/
|-rocksdb/
|-something_else
```

My project name is rocksdb_xxx. While I'm building it at the root path, the cmake file path above is `/path/to/rocskdb_xxx/include/rocksdb/version.h`, which doesn't work. `${CMAKE_CURRENT_SOURCE_DIR}` is a friendly option path, isn't it?

Reply:
i see. presumably you include rocksdb project using `add_subdirectory()` which points `CMAKE_SOURCE_DIR` to `/path/to/rocskdb_xxx`.

Reply:
yeah, that's it.